content/log/explore.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Explore
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+## Overview 
+
+The Log explorer is an optimized view which makes it easy to do any kind of troubleshooting and data exploration.
+
+{{< img src="log/explore/explore_view_with_comments.png" alt="Explore view with comments" >}}
+
+In this view you can:
+
+* [Interact with the Time range](#time-range)
+* [Display logs into lists](#log-list)
+* [Use facets to help you in various investigations](#facets)
```

Comment:
> Use facets to help you in various investigations

I'd like to rephrase this, but I'm not sure how. What investigations?

Reply:
An investigation is being able to filter the log list displayed in order to find the log related only to your issue. ie: fitlering a a given service, on the "error" severity.

Reply:
def updated.